src/main/resources/se/diabol/jenkins/pipeline/DeliveryPipelineView/ComponentSpec/help-lastJob.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    Final job in the pipeline. Usually the build/compile job.
```

Comment:
I don't think this description is accurate. I think it would be sufficient if this just stated "Final job to display in the pipeline view."